test/index.test.ts

```diff
@@ -86,6 +95,24 @@ describe('migrate MongoDB state store', () => {
         done();
       });
     });
+
+    it('returns migrations document from custom collection', async done => {
+      const customCollectionName = '__custom_migrations';
+      const customMigrationDoc = { ...migrationDoc, lastRun: `${new Date().getTime()}-my-migration.js` };
```

Comment:
Create a different object of `migrationDoc` defined above to improve test matching differences.